utils/resizer/src/index.js

```diff
@@ -54,5 +56,6 @@ export function buildResizerURL(
     }
     return _url
   }
-  return null
 }
+
+export { imageANSToImageSrc, signImagesInANSObject, handleFetchError, resizerFetch }
```

Comment:
Just for clarity, can `buildResizerURL` be exported here as well, and the other export keyword on line 21 removed?

Reply:
sure, done in [ede0cde](https://github.com/WPMedia/feed-components/pull/666/commits/ede0cde6833b6ec9c39c94304a7d954e7275838b)